docs/extensions/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Extensions
+has_children: true
+nav_order: 7
+---
+
+# ONNXRuntime-Extensions
+
+[![Build Status](https://dev.azure.com/onnxruntime/onnxruntime/_apis/build/status%2Fmicrosoft.onnxruntime-extensions?branchName=main)](https://dev.azure.com/onnxruntime/onnxruntime/_build/latest?definitionId=209&branchName=main)
+
+## What is ONNXRuntime-Extensions?
+
+Introduction: ONNXRuntime-Extensions is a library that extends the capability of the ONNX models and inference with ONNX Runtime, via ONNX Runtime Custom Operator ABIs. It includes a set of [ONNX Runtime Custom Operators](https://onnxruntime.ai/docs/reference/operators/add-custom-op.html) to support the common pre- and post-processing operators for audio, vision, text, and NLP models. ONNXRuntime-Extensions also supports multiple languages and platforms, like Python on Windows/Linux/macOS, some mobile platforms like Android and iOS, and Web-Assembly etc. The basic workflow is to enhance a ONNX model firstly and then do the model inference with ONNX Runtime and ONNXRuntime-Extensions package.
+
+
+<img src="../../images/combine-ai-extensions-img.png" alt="Pre and post-processing custom operators for vision, text, and NLP models" width="100%"/>
+<sub>This image was created using <a href="https://github.com/sayanshaw24/combine" target="_blank">Combine.AI</a>, which is powered by Bing Chat, Bing Image Creator, and EdgeGPT.</sub>
+
+### Quickstart
+
+## Installation
+### **Python installation**
+```bash
+pip install onnxruntime-extensions
+````
+
+
+### **Nightly Build**
+
+#### <strong>on Windows</strong>
+```cmd
+pip install --index-url https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/pypi/simple/ onnxruntime-extensions
+```
+Please ensure that you have met the prerequisites of onnxruntime-extensions (e.g., onnx and onnxruntime) in your Python environment.
+#### <strong>on Linux/macOS</strong>
+Please make sure the compiler toolkit like gcc(later than g++ 8.0) or clang are installed before the following command
+```bash
+python -m pip install git+https://github.com/microsoft/onnxruntime-extensions.git
+```
+
+## Prepare the pre/post-processing ONNX model
+With onnxruntime-extensions Python package, you can easily get the ONNX processing graph by converting them from Huggingface transformer data processing classes such as in the following example:
```

Comment:
We discussed giving a small example here then referencing the other methods of enhancing the model.

Reply:
right, apologies i missed that! just added some, you can also add any others i might have missed.